lms/djangoapps/certificates/signals.py

```diff
@@ -17,17 +18,18 @@
 from openedx.core.djangoapps.certificates.api import auto_certificate_generation_enabled
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from openedx.core.djangoapps.content.course_overviews.signals import COURSE_PACING_CHANGED
-from openedx.core.djangoapps.signals.signals import COURSE_GRADE_NOW_PASSED, LEARNER_NOW_VERIFIED
+from openedx.core.djangoapps.signals.signals import (COURSE_GRADE_NOW_PASSED, LEARNER_NOW_VERIFIED,
+                                                     USER_CERTIFICATE_DOWNLOADABLE)
 from course_modes.models import CourseMode
 from student.models import CourseEnrollment
 
-
 log = logging.getLogger(__name__)
 CERTIFICATE_DELAY_SECONDS = 2
 
 
 @receiver(COURSE_PACING_CHANGED, dispatch_uid="update_cert_settings_on_pacing_change")
-def _update_cert_settings_on_pacing_change(sender, updated_course_overview, **kwargs):  # pylint: disable=unused-argument
+def _update_cert_settings_on_pacing_change(sender, updated_course_overview,
+                                           **kwargs):  # pylint: disable=unused-argument
```

Comment:
revert this to avoid core change

Reply:
@saad-waqar please be intermediate to resolve the concern.